terraform/environments/bootstrap/single-sign-on/main.tf

```diff
@@ -81,6 +81,13 @@ data "aws_ssoadmin_permission_set" "data_engineer" {
   name         = "modernisation-platform-data-eng"
 }
 
+data "aws_ssoadmin_permission_set" "reporting-operations" {
+  provider = aws.sso-management
+
+  instance_arn = local.sso_instance_arn
+  name         = "mp-reporting-operations"
```

Comment:
This permission set needs to be created here first https://github.com/ministryofjustice/aws-root-account/blob/main/management-account/terraform/sso-admin-permission-sets.tf#L129